h/static/scripts/annotator/plugin/domtextmapper.coffee

```diff
@@ -1,11 +1,26 @@
+# Simple IE autodetect function
+# See for example https://stackoverflow.com/questions/19999388/jquery-check-if-user-is-using-ie/21712356#21712356
+isIE = ->
+  ua = window.navigator.userAgent
+  ua.indexOf("MSIE ") > 0 or      # for IE <=10
+    ua.indexOf('Trident/') > 0 or # for IE 11
+    ua.indexOf('Edge/') > 0       # for IE 12
+
 # Annotator plugin providing dom-text-mapper
 class Annotator.Plugin.DomTextMapper extends Annotator.Plugin
 
   pluginInit: ->
     if @options.skip
-      console.log "Not registering DOM-Text-Mapper."
+      console.log "Not registering DOM-Text-Mapper, since it's disabled."
       return
 
+    unless @options.force
```

Comment:
> Why `skip` if you could just not load the plugin?

I don't remember clearly, but I seem to remember that there is a reason to load the plugin even if we don't plan on using it. Maybe something about declaring a class, or something like that. Or maybe something about the way we loaded plugins made it impossible for us not *not* load it, but it allowed passing in an option to deactivate ...

... I don't remember. It all happened a long time ago. I can refactor.

> Why `force` if the plugin doesn't auto-skip in some edge case?

`Force` is needed to disable the auto-detection. I don't want to *force* the deployers/integrators to use auto-detection.